Modules/_elementtree.c

```diff
@@ -2925,7 +3147,7 @@ expat_set_error(enum XML_Error error_code, Py_ssize_t line, Py_ssize_t column,
     if (errmsg == NULL)
         return;
 
-    error = PyObject_CallFunctionObjArgs(st->parseerror_obj, errmsg, NULL);
+    error = _PyObject_FastCall(st->parseerror_obj, &errmsg, 1);
```

Comment:
FWIW, I'm not sure if these `_PyObject_FastCall()` additions in _elementtree.c should have been made as part of this PR -- they are quite unrelated to the BPO issue.

Reply:
Granted. I made them for the new code I wrote, so I consider it a bit of a consistency cleanup to use them in all similar places. Note that I also tried to apply the general implementation pattern to all the handler functions that either dispatches to the treebuilder or to a Python function. Given how repetitive this code becomes in C, I think avoiding surprises in the implementation is important to simplify the future maintenance.